### ISSUE TITLE
[Important accessibility improvement] Headline structure

### ISSUE THREAD

**Issue by marcus-herrmann** [owncloud/web#2625]
[This is a ticket for the "accessibility backlog"]

Since screen reader users [predominantly navigate with headlines](https://webaim.org/techniques/semanticstructure/#contentstructure) and try to get an overview of a site, document or app by letting their software display all headlines and their inner structure, Phoenix should provide such headlines for structural elements/contents of the site, even when they are visually hidden (using ODS's `oc-visually-hidden`).

**Comment by pascalwengerter** [comment]
Should be resolved in the `a11y-swarming` branch, coming to web `master` soon
